<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>templateSetting</class>
 <widget class="QDialog" name="templateSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>图像平滑模板</string>
   </property>
   <widget class="QRadioButton" name="meanTemplate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>平均模板</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="gaussTemplate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯模板</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="commonTemplate1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>常用模板一</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="commonTemplate2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>常用模板二</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="commonTemplate3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>常用模板三</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
